services/QuillLMS/client/app/bundles/Connect/libs/conceptResults/sharedConceptResultsFunctions.js

```diff
@@ -16,23 +16,22 @@ export function getConceptResultsForAttempt(question, attemptIndex, question_typ
   const prompt = question.prompt.replace(/(<([^>]+)>)/ig, '').replace(/&nbsp;/ig, '');
   const answer = question.attempts[attemptIndex].response.text;
   const attemptNumber = attemptIndex + 1;
+  const { response, } = question.attempts[attemptIndex]
+
   let conceptResults = [];
-  if (question.attempts[attemptIndex].response) {
-    conceptResults = hashToCollection(question.attempts[attemptIndex].response.conceptResults) || [];
-  }
-  if (conceptResults.length === 0 && question_type === 'sentence-fragment-expansion') {
```

Comment:
This is a defunct question type, which actually belonged to the Diagnostic and not to Connect, and we no longer have any questions in the database with this type so we don't need to handle it in the code.